pages/index.tsx

```diff
@@ -1,33 +1,29 @@
 import type { NextPage } from "next";
 import Form from "../components/Form";
-// @ts-ignore
-import MarkdownField from "../components/MarkdownField";
-import useStore from "../zustand/store";
-import styled from "styled-components";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import MarkDownOutput from "../components/MarkdownOutput";
+import WikiUpdater from "../components/WikiUpdater";
 
 const Home: NextPage = () => {
   return (
-    <>
-      <StyledMain>
-        <Styledh1>Github Wiki Generator</Styledh1>
+    <Grid container p={6}>
+      <Grid item xs={12}>
+        <Typography variant="h4" component="h1" my={2} align="center">
```

Comment:
Dont know why you have to go crazy with Typography here, if there is basically only 1 x h1 on the page :D

Reply:
MUI hast the Typography component and I see no reason in not utilizing it. I need to do something with it, why shouldnt I use it? Also Typography is used for the other headings in the app, at at least 3 places.